packages/db/src/provider.ts

```diff
@@ -0,0 +1,59 @@
+import omit from 'lodash/omit';
+import isNil from 'lodash/isNil';
+import isValidHostname from 'is-valid-hostname';
+
+import { ProviderRecord, PostgresProviderRecord } from '@cumulus/types';
+import { PostgresValidationError } from '@cumulus/errors';
+
+import KMS from '@cumulus/aws-client/KMS';
+
+export const encryptValueWithKMS = (
+  value: string
+): Promise<string> => {
+  if (process.env?.provider_kms_key_id === undefined) {
```

Comment:
If we already have a dependency on `cumulus/common` in this package, the `envUtils` has a utility for checking for required vars that is worth using

Reply:
5662de80f